pkg/acsengine/engine.go

```diff
@@ -481,6 +487,11 @@ func (t *TemplateGenerator) getTemplateFuncMap(properties *api.Properties) map[s
 				return ""
 			}
 
+			// add artifacts and addons
```

Comment:
I don't think this is supposed to be here? Duplicate of what's below and `str` isn't used...